src/core/api/index.js

```diff
@@ -91,10 +91,35 @@ export function callApi({
   errorHandler,
   _config = config,
 }: CallApiParams): Promise<any> {
+  if (!endpoint) {
+    return Promise.reject(
+      new Error(`endpoint URL cannot be falsy: "${endpoint}"`)
+    );
+  }
   if (errorHandler) {
     errorHandler.clear();
   }
-  const queryString = makeQueryString({ ...params, lang: state.lang });
+
+  const urlObj = url.parse(endpoint, true);
```

Comment:
Just a nitpick but I like descriptive variable names; we're past the grt Unix vwl shrtg yrs 😉

Eg. could this be `urlObject` instead?

Reply:
Sure, that's fine. I tend to prefer shorter variables because it helps me read the code faster (e.g. less letters to parse). I can change it though.